### PR TITLE
Cookie clicker blockly

### PR DESCRIPTION
**Pull request by jessica-norris**
This PR provides blocks needed for Cookie clicker:

- Javascript blocks to access the DOM and make changes (`querySelector`, set text content, set src) in a new tray called "Script" -- this bit needs design review!
- Rewording and colouring of existing HTML Element blocks after first draft feedback
- Javascript / HTML Code generation fixes:
  - multiple blocks inside the event handler definition block now generate code
  - Core blockly changed the `ORDER_ADDITIVE` to `ORDER_ADDITION`, breaking code gen in some cases, now fixed

This will fix groklearning/issues#380 -- many screenshots of the process inside that issue, including a full cookie clicker functional implementation in the last comment.

@taybenlor I've put you as the reviewer plus Tim as this should clear most of the state off for cookie clicker and I'd like your explicit check of the look of the block trays and colours please.

@timdawborn I feel like there are more potential code gen bugs that I'll find and fix in a separate PR (looks like the python code gen did not rename the constant).

### CODE REVIEW

File: re.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-find . ../blockly \
+find . ../blockly* \
```
Comment (timdawborn) [inline re.sh]: Why was this change needed?

Comment (jessica-norris) [reply re.sh]: This was because I wanted to grep the blockly-blocks code base as well as blockly core. Previously it was only picking up core due to the lack of file name globbing.

Comment (timdawborn) [reply re.sh]: Grepping `blockly-blocks` is what the `.` is for before the `../blockly`

**Comment by taybenlor** [comment]
Removed the `*` - all g?
